Microsoft.Toolkit.Uwp.UI.Controls/ImageEx/ImageExBase.Members.cs

```diff
@@ -80,7 +80,7 @@ public partial class ImageExBase
         /// Gets or sets the CornerRadius for underlying image. <para/>
         /// Used to created rounded/circular images.
         /// </summary>
-        public CornerRadius CornerRadius
+        public new CornerRadius CornerRadius
```

Comment:
Same

Reply:
`new` is needed because we are now replacing the property provided with the new version. We may want to wrap it with disable warning CS0109 for the older versions?